khbr/KH2/KingdomHearts2.py

```diff
@@ -76,11 +76,12 @@ def get_hidden_options(self):
             # utility mod options
             "remove_damage_cap": {"display_name": "Remove Damage Cap", "description": "Removes the damage cap for all enemies in the game.",
                                 "possible_values": [], "hidden_values": [False, True]},
-            "cups_give_xp": {"display_name": "Cups give XP", "description": "You will now gain XP and Form XP for killing enemies in the Olympus cups",
+            "cups_give_xp": {"display_name": "Cups give XP", "description": "You will now gain XP and Form XP for killing enemies in the Olympus cups.",
                                 "possible_values": [], "hidden_values": [False, True]},
-            "retry_data_final_xemnas": {"display_name": "Retry Data Final Xemnas", "description": "If you die to Data Final Xemnas, continue will put you right back into the fight, instead of having to fight Data Xemnas I again (warning will be a softlock if you are unable to beat Final Xemnas)",
+            "retry_data_final_xemnas": {"display_name": "Retry Data Final Xemnas", "description": "If you die to Data Final Xemnas, continue will put you right back into the fight, instead of having to fight Data Xemnas I again.",
                                 "possible_values": [], "hidden_values": [False, True]},
-            "retry_dark_thorn": {"display_name": "Retry Dark Thorn", "description": "If you die to Dark Thorn, continue will put you right back into the fight, instead of having to fight Shadow Stalker again (warning will be a softlock if you are unable to beat Shadow Stalker)",
+            "retry_dark_thorn": {"display_name": "Retry Dark Thorn", "description": "If you die to Dark Thorn, continue will put you right back into the fight, instead of having to fight Shadow Stalker again.",
+            # I removed the mentions of creating an incompletible seed due to the most recent version of auto-save excluding these rooms, kicking you out to Altar of Naught (Final Xemnas) or Hallway (Dark Thorn)
```

Comment:
rejecting this change because auto-save is not bundled with the randomizer, so khbr should not assume users have it. Also it is still a "softlock" because the user has to reload the game, it just isn't something that bricks the seed.